vendor/github.com/getgauge/common/common.go

```diff
@@ -50,7 +50,7 @@ const (
 	ProductName             = "gauge"
 	DotGauge                = ".gauge"
 	config                  = "config"
-	SpecsDirectoryName      = "specs"
+	//SpecsDirectoryName      = "specs"
```

Comment:
It is useful to have this constant. 
Also, In golang vendor folder is used to store external dependencies. So a changes here should not be done.

Reply:
yes - it is intended to have the variable, I used to commented it out to find all usage.
will bring it back in